### ISSUE TITLE
Sonar-Perl 0.5.6 compatibility for SonarQube Enterprise 9.9.0 and 9.9.1

### ISSUE THREAD

**Issue by Monit-dev** [sonar-perl/sonar-perl#140]
Hi,

Please share if the latest perl plugin 0.5.6 is compatible for SonarQube Enterprise 9.9.0 or 9.9.1?

Thanks

**Comment by Monit-dev** [comment]
Hi,

Any update on the above issue?

Thanks

**Comment by otrosien** [comment]
Hi. I haven't touched the project for quite a while... give me a few days to sort things out. 😅

**Comment by otrosien** [comment]
Would you mind trying out 0.6.0? I upgraded some dependencies, and it seems to load, but I haven't made any in-depth investigation of all the features. Beware that because of long-outstanding change https://github.com/sonar-perl/sonar-perl/pull/95 the ID of the plugin changed, so you may have to re-analyse your perl projects from scratch (or investigate how to migrate the data, could be with the help of some sql migration...)

**Comment by Monit-dev** [comment]
Hello Oliver,

I have tried 0.6.0, I see the same result what I had for 0.5.6. It doesn't show any lines of code scanned, zero vulnerabilities even after injecting vulnerabilities in the code.

Thanks

**Comment by otrosien** [comment]
thank you for the feedback. I'll take a look

**Comment by Monit-dev** [comment]
Hi Oliver,

Any updates further regarding my query?

Thanks

**Comment by otrosien** [comment]
Hey Monit-dev, I'm sorry, I couldn't reproduce it yet. Will keep trying but have limited resources for this project at the moment.

**Comment by otrosien** [comment]
I just tried both 9.9 LTS and 10.1 without problems (the sample-project). Can you check your installation if you had two versions of the perl plugin in your extensions folder? Can you paste some relevant lines from your logs that could help investigate the problem?

**Comment by sesa312293** [comment]
Any updates on this issue? Would like to install on our 9.9 LTS server.

**Comment by huntantr** [comment]
https://github.com/integrated-application-development/sonar-delphi

**Comment by cnwaldron** [comment]
I was able to get this working on 10.2.1. However, I ran into the same "issue" where the code was always passing with no issues etc. Then I reviewed the docs again and realized I hadn't seen the part about using perlcritic see at the bottom of the section: [Analyzing your perl project](https://github.com/sonar-perl/sonar-perl/tree/master?tab=readme-ov-file#analyzing-your-perl-project)

After I ran perlcritic with the command at the bottom of the section I now see the results as expected. I hope this helps.
